src/SWP/Bundle/RevisionBundle/SWPRevisionBundle.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Superdesk Publisher Revision Bundle.
+ *
+ * Copyright 2017 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\RevisionBundle;
+
+use SWP\Bundle\StorageBundle\Drivers;
+use SWP\Bundle\StorageBundle\DependencyInjection\Bundle\Bundle;
+
+class SWPRevisionBundle extends Bundle
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getSupportedDrivers()
+    {
+        return [
+            Drivers::DRIVER_DOCTRINE_ORM,
+        ];
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getModelClassNamespace()
+    {
+        return 'SWP\Bundle\RevisionBundle\Model';
```

Comment:
it is, but `PersistableInterface` is implemented in Bundle so it's registered in it.